test/quantization/test_numeric_suite_fx.py

```diff
@@ -771,3 +771,151 @@ def forward(self, x):
             mp_shadows_mq, OutputLogger)
         self.assertTrue(len(act_compare_dict) == 2)
         self.assert_ns_logger_act_compare_dict_valid(act_compare_dict)
+
+class TestFXNumericSuiteCoreAPIsModels(QuantizationTestCase):
+    """
+    Tests numeric suite core APIs on non-toy models.
+    """
+
+    def _get_sparsenn_toy_model(self):
+        class DenseTopMLP(nn.Module):
```

Comment:
NIT: Can we move all models to https://github.com/pytorch/pytorch/blob/master/torch/testing/_internal/common_quantization.py#L865